nifi-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -60,16 +60,10 @@ dataflows.
 
 WARNING: Before proceeding, check the Admin Guide to confirm you have the <<administration-guide.adoc#system_requirements,minimum system requirements>> to install and run NiFi.
 
-NiFi can be downloaded from the link:http://nifi.apache.org/download.html[NiFi Downloads page^]. There are two packaging options
-available:
+NiFi can be downloaded from the link:http://nifi.apache.org/download.html[NiFi Downloads page^]. After downloading the version of NiFi
```

Comment:
The link can be updated:
```suggestion
NiFi can be downloaded from the link:https://nifi.apache.org/download/[NiFi Downloads page^]. After downloading the version of NiFi
```

Reply:
Thanks for catching this, I corrected two other places where nifi.apache.org URLs were used.  Update committed.